chart/varnish.tpl

```diff
@@ -0,0 +1,61 @@
+vcl 4.0;
+
+import std;
+import directors;
```

Comment:
Having an example for the configuration template is a sensible idea, but I'm not sure if it brings a lot value to ship it with the chart (and even use it as default), since it most probably _will_ need customization by the user, and there's no real way to overwrite this file when installing the chart.

Another way to go at this might be to expect the user to provide their own "install chart" (with this one as dependency) and just add some documentation to the README on how to achieve this.

On a side note, I suspect this might be a good use case for helm/helm#3276.

Reply:
moved to values.yaml